executor/utils.go

```diff
@@ -13,15 +13,23 @@
 
 package executor
 
-import "strings"
+import (
+	"strings"
+)
 
-func setFromString(value string) []string {
+// SetFromString constructs a slice of strings from a comma separated string.
+// It is assumed that there is no duplicated entry. You could use addToSet to maintain this property.
+func SetFromString(value string) []string {
```

Comment:
It's not used outside this package, why it's exported?

Reply:
Exported for tests.

Tests in executor is all named `_test`, thus tests are treated as a standalone package, which means they can not invoke internal/unexported functions. A [link](https://github.com/pingcap/tidb/pull/23519) for that.

We may solve it by:

1. Copy `setFromString` to tests(it is a simple helper, but I want DRY)
2. Move the tests into a non-`_tests` package(Don't want to extract these tests, )
3. Export `setFromString`